backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -28,7 +28,7 @@ object BackendJobExecutionActor {
 /**
   * Workflow-level actor for executing, recovering and aborting jobs.
   */
-trait BackendJobExecutionActor extends BackendLifecycleActor with ActorLogging {
+trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLogging {
```

Comment:
Ha, in fact it was already in the trait. I think the method-level `jobDescriptor`s were just left over from when it was workflow scoped